compiler/noirc_frontend/src/hir/comptime/interpreter/foreign.rs

```diff
@@ -18,6 +21,7 @@ pub(super) fn call_foreign(
 ) -> IResult<Value> {
     match name {
         "poseidon2_permutation" => poseidon2_permutation(interner, arguments, location),
+        "keccakf1600" => comptime_keccakf1600(interner, arguments, location),
```

Comment:
Is this different from the normal keccakf1600? If not we should name them the same like we do for poseidon2_permutation. I'd prefer to qualify `acvm::blackbox_solver::keccakf1600` if needed

Reply:
> I'd prefer to qualify acvm::blackbox_solver::keccakf1600 if needed

No it isn't, it was just for naming. I'll just do this suggestion.